src/styles/main.scss

```
@@ -1 +1,71 @@
 // put code here
+body {
+  min-height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.stopwatch {
+  width: 80vmin;
+  height: 80vmin;
+  border-radius: 50%;
+  border: 8px dashed #333;
+  position: relative;
+}
+
+.stopwatch::after {
+  content: "";
+  width: 50px;
+  height: 50px;
+  border-radius: 50%;
+  background-color: #821b1b;
+  z-index: 100;
+
+}
```

```suggestion
  z-index: 1;
}
```
There is no need to increase z-index so much. It may be difficult to override this value later